containers/ecr-viewer/src/app/api/migrate-db/migrations/core/index.ts

```diff
@@ -2,5 +2,7 @@ export default {
   "19700101000000_initial.ts": require("./19700101000000_initial"),
   "20250425111400_user_prog.ts": require("./20250425111400_user_prog"),
   "20250428110000_condition_ref.ts": require("./20250428110000_condition_ref"),
-  "20250505100700_ecr_fks.ts": require("./20250505100700_ecr_fks"),
+  "20250509000000_condition_fk.ts": require("./20250509000000_condition_fk"),
+  "20250509000001_ecr_ref.ts": require("./20250509000001_ecr_ref"),
```

Comment:
this name needs to be changed back to the original - it'll mess with kyseley's tracking to change it